### ISSUE TITLE
[1.7.10] Levels r3.0.1 Ticking entity crash

### ISSUE THREAD

**Issue by lev4ukpavel2** [TheXFactor117/Levels#140]
https://pastebin.com/BqMS4Ac6

**Comment by FN343** [comment]
Yeah, I've been getting a similar crash whenever I try to kill a mob with TiC Weapons (Mainly the crossbow's bolts and sometimes the cleaver). This used to not happen until I updated FastCraft and Optifine recently, but that shouldn't be the reason why it would crashing and it doesn't say so in the crash report. Here's the link to my issue I posted as well:

[Issue #153](https://github.com/TheXFactor117/Levels/issues/153)
